target/linux/ath79/dts/ar7161_netgear_wndr3800.dts

```diff
@@ -193,6 +227,10 @@
 	pll-data = <0x11110000 0x00001099 0x00991099>;
 
 	mtd-mac-address = <&art 0x00>;
+	mtd-mac-address-local-admin;
+
+	resets = <&rst 9>;
+	reset-names = "mac";
 
```

Comment:
> (* additionally, phy reset removed from eth0)
PR #1252 should fix the switch issue due to resets. If that PR is accepted the two reset statementes can be dropped as that is changed in ar7100.dtsi.

Reply:
I know, I just did it also here just to make the current version available.
I will rebase my PR if that PR gets accepted first